auth/auth/driver/driver.py

```diff
@@ -275,14 +279,14 @@ def secret_data(self):
         config = SQLConfig(
             host=server_config.host,
             port=server_config.port,
-            user=self.name if CLOUD != 'azure' else f'{self.name}@{server_config.instance}',
+            user=self.name,
             password=self.password,
             instance=server_config.instance,
             connection_name=server_config.connection_name,
             db=self.name,
-            ssl_ca='/sql-config/server-ca.pem',
-            ssl_cert='/sql-config/client-cert.pem',
-            ssl_key='/sql-config/client-key.pem',
+            ssl_ca=server_config.ssl_ca,
```

Comment:
This might actually be very incorrect if the current config is mounted as database-server-config which it looks like it is.

Reply:
Can you describe why this change is necessary?

These paths are hardcoded into the MYSQL config file, meaning we can't arbitrary relocate the secret. That was frustrating when I first built this stuff. In the end, I embraced the fixed location of the secret and that seemed to mostly avoid issues.

Reply:
I think I was trying to lump in the optionality of the `ssl_cert`/`ssl_key` fields with what I thought was some needless repetition, but I see now that what I replaced these literals with are not necessarily the same. I reverted this but think I do need to have the conditional on whether to make `ssl_cert` and `ssl_key` None.